onadata/apps/api/viewsets/floip_viewset.py

```diff
@@ -43,6 +45,39 @@ def build_json_resource_obj(cls, fields, resource, resource_instance,
         return obj
 
 
+class FLOIPResponsePageNumberPagination(JsonApiPageNumberPagination):
+
+    def get_paginated_response(self, data, descriptor_id: str):
+        nxt = None
+        previous = None
+
+        if self.page.has_next():
+            nxt = self.page.next_page_number()
```

Comment:
Can we rename this to `next`; for name consistency

Reply:
`next` is a python keyword. Pylint/Codacy complains when it's used... I'll update this and the `previous` variable to `(next|previous)_page_number`

Reply:
Changed in the latest commit